### PR TITLE
Making sure that builds of side tags update are marked as signed.

### LINKED ISSUE #4032: Side tag updates are not automatically pushed to testing
When I create a side tag update for a stable Fedora, it sits in pending until I manually click "push to testing". This is inconsistent with regular updates and a tad confusing.

See also https://pagure.io/fedora-infrastructure/issue/8930

### PR DESCRIPTION
**Pull request by cverna**
Fixes #4032.

Signed-off-by: Clement Verna <cverna@tutanota.com>

### CODE REVIEW

File: bodhi/server/consumers/signed.py [hunk 1]
```diff
@@ -89,7 +89,9 @@ def __call__(self, message: fedora_messaging.api.Message):
                 log.info('Build is not assigned to release, skipping')
                 return
 
-            if build.update and build.update.from_tag:
+            if build.update and \
+               build.update.from_tag and not \
+               build.update.release.composed_by_bodhi:
```
Comment (pypingou) [inline bodhi/server/consumers/signed.py]: For readability I tend to put the and on the new line and I would encourage it even more for the ``not`` as it is pretty important :)

**Comment by pypingou** [comment]
One nitpick, let me know what you think of it, happy to approve afterward

File: bodhi/server/consumers/signed.py (hunk 1 above)
Comment (cverna) [reply bodhi/server/consumers/signed.py]: updated :)

**Review by pypingou — APPROVED** [review]
Looking good!

**Pull request merged**